src/packages.jl

```diff
@@ -453,6 +453,69 @@ function build_recursive(name::String; quiet::Bool = false,
   return true
 end
 
+"""
+    test(name::String)
+
+Return a boolean indicating if the GAP package with name `name` succeeds
+in running its tests.
+
+It is inteded to be used with the `@test` macro from the `Test` package.
+
+The function uses [the function `TestPackage`](GAP_ref(ref:TestPackage)).
+"""
+function test(name::String)
+  global disable_error_handler
+
+  function with_gap_var(f, name::String, val)
+    gname = GapObj(name)
+    old_value = Globals.ValueGlobal(gname)
+    Globals.MakeReadWriteGlobal(gname)
+    Globals.UnbindGlobal(gname)
+    Globals.BindGlobal(gname, val)
```

Comment:
I think this would also work?
```suggestion
    replace_global!(gname, val)
```